src/edu/ucalgary/ensf409/Filing.java

```diff
@@ -0,0 +1,80 @@
+package edu.ucalgary.ensf409;
+
+public class Filing {
+
+	
+	
+	private String iD;
```

Comment:
Filings should extend Furniture, Therefore you dont need the private stirngs for ID, ManuID, type etc. I think if you follow the same format as mackenzie's chair class it should be fine